ext/psych/psych_parser.c

```diff
@@ -160,7 +160,7 @@ static VALUE transcode_io(VALUE src, int * parser_encoding)
 	return src;
     }
 
-    /* Just guess on ASCII-8BIT */
+    /* Just guess on BINARY */
     if (io_external_enc_index == rb_ascii8bit_encindex()) {
```

Comment:
Does this proposal keep `ascii8bit` name in  C API?

Reply:
Yes, we could alias it as well, but since C-extensions are usually written by more advanced users it's not that useful I think.

Reply:
I would advocate for consistency across C and ruby